cypress/integration/scaffolded-question.test.js

```diff
@@ -79,6 +79,71 @@ context("Test open response interactive", () => {
               answerType: "open_response_answer",
               answerText: "Test subquestion answer"
             }
+          },
+          answerText: "[Level: 1] Test subquestion answer"
+        });
+      });
+    });
+
+    it("logs events from subinteractives to parent", () => {
+      phonePost("initInteractive", {
+        mode: "runtime",
+        authoredState: {
+          version: 1,
+          prompt: "Test prompt",
+          hint: "Hint",
+          subinteractives: [
+            {
+              id: "int1",
+              url: "/open-response",
+              authoredState: {
+                version: 1,
+                prompt: "Subquestion prompt",
+                hint: "Subquestion hint",
+                questionType: "open_response"
+              }
+            }
+          ]
+        }
+      });
+      phoneListen("log");
+
+      cy.getNestedIframeBody().find("textarea").type("Test subquestion answer");
+      cy.focused().blur();
+      cy.wait(200);
+
+      getAndClearAllPhoneMessage((messages) => {
+        expect(messages.length).eql(3);
+
+        expect(messages[0]).eql({
+          action: "focus in",
+          data: {
+            focus_target: "textarea",
+            scaffolded_question_level: 1,
+            subinteractive_url: "/open-response",
+            subinteractive_type: "open_response",
+            subinteractive_sub_type: undefined
```

Comment:
I thought mostly about interactives like fill in the blank. Its type is "iframe_interactive", so it's not very descriptive (since we use the same metadata here). sub_type will be "fill_in_the_blak" (I don't think it's set yet), so it should make it a bit easier to filter events and provide more context. Does it make sense?
And right, I'll add subinteractive_id.

Reply:
Yeah, that makes sense. So it is fine to leave it.